src/client/dfs/SConscript

```diff
@@ -46,18 +46,17 @@ def scons():
     dfs_internal_src = ['dfs_internal.c']
     dfs_internal = daos_build.library(denv, 'dfs_internal',
                                       dfs_internal_src, LIBS=libraries)
-
     denv.Install('$PREFIX/lib64/', dfs_internal)
 
+    duns = daos_build.library(denv, 'duns', 'duns.c', LIBS=libraries)
+    denv.Install('$PREFIX/lib64/', duns)
+
     fenv = denv.Clone()
     prereqs.require(fenv, 'fuse')
     dfuse_hl_src = ['dfuse_hl.c']
+    libraries += ['fuse3']
     dfuse_hl = daos_build.program(fenv, 'dfuse_hl', dfuse_hl_src,
```

Comment:
this line is still building dfuse_hl, no?

Also, you need to remove dfuse_hl from utils/rpms/daos.spec and debian (and add change log)

Reply:
yea i had removed then re-added it.
i want to build it but not install it.
i see rpm build failing, thanks for the hint.

Reply:
i guess maybe just not build it is also fine.. sorry im going back and forth on this.

Reply:
I would be in favor of removing it ...